qiskit-superstaq/qiskit_superstaq/validation.py

```diff
@@ -1,4 +1,3 @@
-import re
```

Comment:
Seems sort of weird to have a lone function in this file and not move all of the validation code to gss. I assume this is because we don't want to have qiskit specific functions in gss?

Reply:
yep - it requires qiskit, which we don't want to add as a dependency for gss

i agree it's a little weird, but i couldn't come up with a less weird place to put it - we use it in more than one place in qss so it doesn't really make sense in any of the existing files, and it requires qiskit which we don't want to add as a dependency for gss. i eventually justified this to myself by assuming we'll probably have other qiskit-specific validators in the future